specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/examples/CreateKafkaClusterWithKafkaRestProxy.json

```diff
@@ -0,0 +1,223 @@
+{
+  "parameters": {
+    "clusterName": "cluster1",
+    "resourceGroupName": "rg1",
+    "api-version": "2018-06-01-preview",
+    "subscriptionId": "subid",
+    "parameters": {
+      "properties": {
+        "clusterVersion": "4.0",
+        "osType": "Linux",
+        "tier": "Standard",
+        "clusterDefinition": {
+          "kind": "kafka",
+          "componentVersion": {
+            "Kafka": "2.1"
+          },
+          "configurations": {
+            "gateway": {
+              "restAuthCredential.isEnabled": true,
+              "restAuthCredential.username": "admin",
+              "restAuthCredential.password": "**********"
```

Comment:
While this is an example, please ensure that secrets (Password) are never returned in a GET, as suggested here

Reply:
> While this is an example, please ensure that secrets (Password) are never returned in a GET, as suggested here

Hi @KrisBash  Thanks for your comment. We have user role to control this.